test/Unit/SqlInputBindingTests.cs

```diff
@@ -60,17 +61,16 @@ public void TestNullCommand()
         [Fact]
         public void TestNullArgumentsSqlAsyncEnumerableConstructor()
         {
-
             Assert.Throws<ArgumentNullException>(() => new SqlAsyncEnumerable<string>(connection, null));
             Assert.Throws<ArgumentNullException>(() => new SqlAsyncEnumerable<string>(null, new SqlAttribute("")));
         }
 
+        /// SqlAsyncEnumerable should throw InvalidOperationExcepion when invoked with an invalid connection 
```

Comment:
Use actual XML doc (with <summary> tags) here. And I would probably specify that this is testing with an empty connection string setting (since invalid could also mean non-empty but also non-existent setting)

Reply:
No other places require the connection open check, am going to get this in and we can prep for a release.